content/en/docs/test.md

```diff
@@ -57,79 +57,99 @@ culpa qui officia deserunt mollit anim id est laborum.
 - Bullet lists can have paragraphs or block elements within them.
 
-  Just indent the content to be even with the text of the bullet point, rather
-  than the bullet itself.
+      Just indent the content to be one tab stop beyond the text of the bullet
+      point. **This paragraph and the code block line up with the second `l` in
+      `Bullet` above.**
 
-  ```bash
-  ls -l
-  ```
+      ```bash
+      ls -l
+      ```
   
-  - And a sub-list after some block-level content
+      - And a sub-list after some block-level content
+    
 - A bullet list item can contain a numbered list.
-  1.  Numbered sub-list item 1
-  2.  Numbered sub-list item 2
+    1.  Numbered sub-list item 1
+    2.  Numbered sub-list item 2
 
 ### Numbered lists
 
 1.  This is a list item
 2.  This is another list item in the same list. The number you use in Markdown
     does not necessarily correlate to the number in the final output. By
     convention, we keep them in sync.
-3.  For single-digit numbered lists, using two spaces after the period makes
+3.  {{<note>}}
+    For single-digit numbered lists, using two spaces after the period makes
     interior block-level content line up better along tab-stops.
+    {{</note>}}
 
+<!-- separate lists -->
 
-
-1.  This is a new list. It has two blank lines from the previous list, If it only
-    had one blank line, it would still show up as part of the previous list.
+1.  This is a new list. With Hugo, you need to use a HTML comment to separate
+    two consecutive lists. **The HTML comment needs to be at the left margin.**
 2.  Numbered lists can have paragraphs or block elements within them.
 
-    Just indent the content to be even with the text of the bullet point, rather
-    than the bullet itself.
+      Just indent the content to be one tab stop beyond the text of the bullet
+      point. **This paragraph and the code block line up with the `m` in
+      `Numbered` above.**
 
-    ```bash
-    $ ls -l
-    ```
-  
-  - And a sub-list after some block-level content
+      ```bash
+      $ ls -l
+      ```
+      
+    - And a sub-list after some block-level content. This is at the same
+      "level" as the paragraph and code block above, despite being indented
+      more.
 
 
 ### Checklists
 
+Checlists are technically bullet lists, but the bullets are suppressed by CSS.
+
 - [ ] This is a checklist item
 - [x] This is a selected checklist item
 
 ## Code blocks
 
-You can create code blocks two different ways:
-
-- by indenting the entire code block 4 spaces from the content
-
-      this is a code block created by indentation
-      
-- by surrounding the code block with three back-tick characters on lines before
-  and after the code block.
+You can create code blocks two different ways by surrounding the code block with
+three back-tick characters on lines before and after the code block. **Don't use
+the indentation method of creating code blocks, because it does not work
```

Comment:
Replace this sentence with:
```
Only use back-ticks for code blocks. The indentation method of creating code blocks does not work reliably in all situations.
```
So:
> Only use back-ticks for code blocks. The indentation method of creating code blocks does not work reliably in all situations.

Reply:
Is this just about the line length?

Regardless, there are one [specific case](https://github.com/russross/blackfriday/issues/239) in which you can't use backticks to denote a code block in Hugo, regardless of whether your code block is in a list or not. It's a Black Friday bug and we are going to hit it a lot because of all our YAML. Not sure what to do about it because Black Friday bugs don't seem to always get fixed very quickly.  That's been open since Feb 2016.